python_tests/pytest_fusion_definitions.py

```diff
@@ -0,0 +1,73 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# Owner(s): ["module: nvfuser"]
+
+import torch
+
+from pytest_core import OpInfo
+
+from nvfuser import FusionDefinition
+from nvfuser.pytorch_utils import (
+    python_scalar_to_nvfuser_dtype,
+    torch_dtype_to_nvfuser_dtype,
+)
+
+
+def parse_inputs_fusion_definition(fd: FusionDefinition, opinfo: OpInfo, *args):
+    if len(args) == 0:
+        return []
+
+    nvf_args = []
+
+    if opinfo.symbolic_parameter_list is None:
+        opinfo.symbolic_parameter_list = [ArgumentType.Symbolic] * len(args)
+    assert len(opinfo.symbolic_parameter_list) == len(args)
+
+    for arg_type, a in zip(opinfo.symbolic_parameter_list, args):
+        if arg_type == ArgumentType.Symbolic:
+            if type(a) is torch.Tensor:
```

Comment:
Nit: i usually use `isinstance(a, torch.Tensor)` for such checks since it will also return true for subclass objects, unless I have a specific reason to match the class exactly.